src/hera.cpp

```diff
@@ -357,6 +359,20 @@ int evm_set_option(
     return 1;
   }
 #endif
+  if (strcmp(name, "vm") == 0) {
+    if (strcmp(value, "binaryen") == 0)
```

Comment:
Could use a `map<string,hera_vm>` and that would simplify this a lot.

Reply:
Wouldn't populating the map be just as tedious?